packages/nextjs/src/config/loaders/dataFetchersLoader.ts

```diff
@@ -170,12 +171,23 @@ export default function wrapDataFetchersLoader(this: LoaderThis<LoaderOptions>,
       path.relative(projectDir, this.resourcePath),
     );
 
+    // Get the route name from this page's filepath
+    let route = this.resourcePath.match(pageRegex)?.[2];
+    if (!route) {
+      logger.warn(`Unable to wrap code from page ${this.resourcePath}, because the route regex has no matches.`);
```

Comment:
Just curious, in what cases does this happen?

Reply:
It shouldn't ever, really. I should just `!` that and explain why.

Reply:
Update: In ended up changing my approach, so a regex isn't even needed here, and `route` will always exist.